src/Components/test/testassets/BasicTestApp/RouterTest/TestRouterWithOnNavigate.razor

```diff
@@ -19,8 +19,9 @@
 @code {
     private Dictionary<string, Func<NavigationContext, Task>> preNavigateTasks = new Dictionary<string, Func<NavigationContext, Task>>()
     {
-        { "LongPage1", new Func<NavigationContext, Task>(TestLoadingPageShows) },
-        { "LongPage2", new Func<NavigationContext, Task>(TestOnNavCancel) }
+    { "LongPage1", new Func<NavigationContext, Task>(TestLoadingPageShows) },
+    { "LongPage2", new Func<NavigationContext, Task>(TestOnNavCancel) },
+    { "Other", new Func<NavigationContext, Task>(TestOnNavException) }
```

Comment:
nit: Formatting

Reply:
Thanks! This is the price I pay for doing most of my dev work in VS Code.